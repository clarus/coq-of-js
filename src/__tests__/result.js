// @flow
import {compileAndPrint} from "../compiler/index.js";

it("can handle several errors", () => {
  expect(compileAndPrint(`const a1 = [,]; const a2 = [,];`)).toMatchSnapshot();
});
