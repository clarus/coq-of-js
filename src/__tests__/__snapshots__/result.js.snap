// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can handle several errors 1`] = `
"> 1 | const a1 = [,]; const a2 = [,];
    |           ^^^

Expected non-empty elements in the array

**********************

> 1 | const a1 = [,]; const a2 = [,];
    |                           ^^^

Expected non-empty elements in the array"
`;
